server/lib/calendar/calendar.findCurrentlyRunningEvent.js

```diff
@@ -43,6 +43,7 @@ async function findCurrentlyRunningEvent(calendars, calendarEventNameComparator,
   switch (calendarEventNameComparator) {
     // do nothing in that case
     case 'has-any-name':
+    case 'does-not-contains':
```

Comment:
Why didn't you do the opposite of contains?

```
queryParams.where.name = {
        [Op.notLike]: `%${calendarEventName}%`,
      };
```

Reply:
It was my first implementation, but the case where we have two events with:
- event 1 with calendarEventName in name
- event 2 without calendarEventName in name
will continue the scene (because at least one event 2 found) and it's not what we want.

Reply:
Oh yes indeed, you're right 🫣